devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp

```diff
@@ -403,6 +403,14 @@ void YarpRobotLoggerDevice::run()
         }
     }
 
+    for (const auto& sensorname : m_robotSensorBridge->getTemperatureSensorsList())
+    {
+        if (m_robotSensorBridge->getTemperature(sensorname, m_ftTemperatureBuffer))
+        {
+            m_bufferManager.push_back({m_ftTemperatureBuffer}, time, "temperatures::" + sensorname);
+        }
+    }
+
```

Comment:
We need to consider the comment from @GiulioRomualdi here, https://github.com/ami-iit/bipedal-locomotion-framework/pull/519#issuecomment-1080436810

I was quickly skimming the code of the  [`embObjFTSensor`](https://github.com/robotology/icub-main/tree/26c74355c9d6463762352f41f8f1a1df593844ca/src/libraries/icubmod/embObjFTsensor) yesterday, although I did not come to a complete understanding, I think retrieving the temperature data is done through service calls from the low level. 

cc @traversaro could you shed more light on this, if you know some details about this sensor?

Reply:
You can call the temperature method anytime you want, as you can see in https://github.com/robotology/icub-main/blob/26c74355c9d6463762352f41f8f1a1df593844ca/src/libraries/icubmod/embObjFTsensor/embObjFTsensor.cpp#L419 the method just access a buffer.